tools/src/check-packages/checkPackageAsync.ts

```diff
@@ -13,22 +13,29 @@ const { green } = chalk;
  */
 export default async function checkPackageAsync(
   pkg: Package,
-  options: ActionOptions
+  options: ActionOptions & { isPlugin?: boolean }
```

Comment:
```suggestion
  options: ActionOptions
```
Add this flag to the `ActionOptions` type in `./types` file 🥸